src/main/java/seedu/address/model/ModelManager.java

```
@@ -144,6 +144,16 @@ public boolean hasTutorial(Tutorial t) {
         return addressBook.hasTutorial(t);
     }
 
+    @Override
+    public void addAttendance(Tutorial t, Student s) {
+        addressBook.addAttendance(t, s);
+    }
+
+    @Override
+    public void markAttendance(Tutorial t, int w, Student s) throws DuplicateItemException, ItemNotFoundException {
```

Do u need to throw these exception, I don't see `AddressBook#markAttendance` throwing any error